packages/terra-date-picker/package.json

```diff
@@ -21,6 +21,7 @@
   },
   "homepage": "https://github.com/cerner/terra-core#readme",
   "devDependencies": {
+    "react-intl": "^2.3.0",
```

Comment:
This should probably be a peerDependency.  The component depends on react-intl but we want to allow the consumer to choose the version of react-intl.